VideoApi/VideoApi.IntegrationTests/Database/Commands/UpdateConferenceParticipantsCommandTests.cs

```diff
@@ -135,16 +139,20 @@ public async Task Should_update_existing_participants_in_conference_and_remove_a
             var updatedParticipantsLinkedParticipant = conference.Participants.SingleOrDefault(x => x.Id == participantOnesLinkedParticipant.Id);
 
             //Assert
-            updatedParticipant.ContactEmail.Should().Be(participantOne.ContactEmail);
-            updatedParticipant.ContactTelephone.Should().Be(participantOne.ContactTelephone);
             updatedParticipant.DisplayName.Should().Be(participantOne.DisplayName);
-            updatedParticipant.FirstName.Should().Be(participantOne.FirstName);
-            updatedParticipant.LastName.Should().Be(participantOne.LastName);
             updatedParticipant.Name.Should().Be(participantOne.Name);
-            updatedParticipant.Representee.Should().Be(participantOne.Representee);
             updatedParticipant.Username.Should().Be(participantOne.Username);
             updatedParticipant.LinkedParticipants.Should().BeEmpty();
             updatedParticipantsLinkedParticipant.LinkedParticipants.Should().BeEmpty();
+
+            if (participantOne is Participant participantCasted)
+            {
+                ((Participant)updatedParticipant).ContactEmail.Should().Be(participantCasted.ContactEmail);
```

Comment:
can we not just cast updatedParticipant as Participant and then we wont have to do cast it every time? so something like this wherever updatedParticipant is being returned? 
var updatedParticipant = result.Value as Participant; or  var updatedParticipant = (Participant)result.Value; 

Although the line where updatedParticipant is being casted is hidden so I am not sure

Reply:
This has been changed for the test.